Common/src/main/java/jeresources/jei/mob/MobWrapper.java

```diff
@@ -77,6 +86,8 @@ public void drawInfo(int recipeWidth, int recipeHeight, @NotNull PoseStack poseS
 
         Font.normal.print(poseStack, this.mob.getLightLevel().toString(), 7, 22);
         Font.normal.print(poseStack, TranslationHelper.translateAndFormat("jer.mob.exp") + ": " + this.mob.getExp(), 7, 32);
+        
+        RenderHelper.drawTexture(poseStack, 150, 18, 22, 120, 18, 18, Resources.Gui.Jei.VILLAGER.getResource());
```

Comment:
Why are we drawing texture in `drawInfo` and why is it the VILLAGER texture?

Reply:
Removed "RenderHelper.drawTexture(..." updated mob.png instead